open_buildings/download_buildings.py

```diff
@@ -130,8 +133,76 @@ def quad2json(quadkey_input):
     click.echo(json.dumps(result, indent=2))
 
 
-def download(geojson_input, format, generate_sql, dst, silent, overwrite, verbose, data_path, hive_partitioning, country_iso):
-
+def download(
+        geojson_data: Dict[str, Any], 
+        dst: Path | str = "buildings.json",
+        source: Source | str = Source.OVERTURE,
+        format: Optional[Format | str] = None, 
+        country_iso: Optional[str] = None,
+        *,
+        generate_sql: bool = False, # whether to actually perform actions or just generate sql
+        verbose: bool = False, # print detailed logs, use python logger instead!
+        silent: bool = False, # no log output, use python logger instead!
+        overwrite: bool = False # whether to overwirte existing output file
```

Comment:
```suggestion
        overwrite: bool = False # whether to overwrite existing output file
```

Reply:
got a docs build failure from an aggressive spell checking, so just fixing this up.